README.md

```diff
@@ -17,10 +17,20 @@ Running a local development environment
 
 If you use Visual Studio Code, just open this repository.
 
+Steps for Mac:
+
 1. First, start the database servers. Choose Tasks > Run Task > "Run Database Servers"
 1. If this is the first time running BORIS, apply the migrations: Choose Tasks > Run Task >
-   "Apply database migrations".
+   "Apply database migrations"
 1. Hit F5 to run the backend server
-1. Choose Tasks > Run Build Task to build the frontend (and re-build it upon any changes).
+1. Choose Tasks > Run Build Task to build the frontend (and re-build it upon any changes)
+
+Steps for Windows 10:
+
+1. Open the project's backend folder with the Docker Quickstart Terminal and run `docker-compose up`
```

Comment:
Is there something Windows-specific I can change in [tasks.json](https://github.com/from-the-storm/BORIS/blob/master/.vscode/tasks.json#L23-L33) so that this can use the same instructions as above?

Reply:
Yeah probably but in either case it still seems to require that I start up the Docker Quickstart Terminal. Maybe we can try some things in person when we meet up next? It really doesn't take me very long though to do it this way so it might be more effort than it's worth.

Reply:
Ok let's check in person.